tests/on-device/config-h-m.bats

```diff
@@ -2,6 +2,16 @@
 
 load lib
 
+setup() {
+  _setup
+  cp ~/.nix-channels ~/.nix-channels.bak
+}
+
+teardown() {
+  nix-channel --remove home-manager
+  mv ~/.nix-channels.bak ~/.nix-channels
```

Comment:
Why both?

Reply:
`nix-channel --remove home-manager` removes link in `~/.nix-defexpr`, so that the NIX_PATH is predictable again.

`mv ~/.nix-channels.bak ~/.nix-channels` just ensures that other channels configured before are restored, so that your installation is not messed up and can be used normally afterwards.